Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/datadonation/ui/DataDonationTestFragmentViewModel.kt

```diff
@@ -32,6 +41,21 @@ class DataDonationTestFragmentViewModel @AssistedInject constructor(
     val errorEvents = SingleLiveEvent<Throwable>()
     val copyJWSEvent = SingleLiveEvent<String>()
 
+    val otp: String = otpRepository.otp?.toString() ?: "No OTP received yet"
```

Comment:
We should show the `OTPAuthorizationResult` here instead. The otp will be deleted after it got authenticated. And since the OTP is always authenticated after it generation, it will always be null when this screen is opened.

Reply:
was not there when I made it yesterday